src/js/Controllers/BCController.js

```diff
@@ -86,7 +86,13 @@ class BCController {
         switch (methodName) {
             case "SetAppProperties":
                 let success = (rpc.result.resultCode == 'SUCCESS')
-                let entry = store.getState().appStore.appsPendingSetAppProperties[0]
+                let appsPendingSetAppProperties = store.getState().appStore.appsPendingSetAppProperties;
+                if(!appsPendingSetAppProperties || appsPendingSetAppProperties.length == 0){
+                    console.error("SetAppProperties Response: no apps in pending queue");
```

Comment:
was this case hit? this shouldn't be possible.

Reply:
Just a null check. I wasn't able to hit this case when I tested but there might be a way to trigger it